lib/fluent/plugin/filter_sanitizer.rb

```diff
@@ -23,15 +23,18 @@ class SanitizerFilter < Fluent::Plugin::Filter
 
       helpers :event_emitter, :record_accessor
 
-      desc "Hash salt to be used to generate hash values with MD5(optional)"
+      desc "Hash salt to be used to generate hash values with specified hash(optional)"
       config_param :hash_salt, :string, default: ""
-      
-      config_section :rule, param_name: :rules, multi: true do 
+
+      desc "Hash scheme to use for generating hash value (supported schemes are md5,sha1,sha256,sha384,sha512) (optional)"
+      config_param :hash_scheme, :string, default: "md5"
```

Comment:
`enum` type is better. No need manual check.

https://docs.fluentd.org/plugin-development/api-config-types#enum